docs/en/stack/ml/df-analytics/dfa-classification.asciidoc

```diff
@@ -4,70 +4,100 @@
-{classification} algorithm: numerical, categorical, and boolean. Arrays are not 
-supported in the {feature-var} fields.
+{classification-cap} is a {ml} process that enables you to predict the class or
+category of a data point in your data set. Typical examples of {classification}
+problems are predicting loan risk, classifying music, or detecting cancer in a
+DNA sequence. In the first case, for example, the data set might contain the
+investment history, employment status, debit status, and other financial details
+for loan applicants. Based on this data, you could use {classanalysis} to create
+a model that predicts whether it is safe or risky to lend money to applicants.
+In the second case, the data contains song details that enable you to classify
+music into genres like hip-hop, country, or classical, for example.
+{classification-cap} is for predicting discrete, categorical values, whereas
+<<dfa-regression,{reganalysis}>> predicts continuous, numerical values.
+
+When you create a {classification} job, you must specify which field contains 
+the classes that you want to predict. This field is known as the _{depvar}_. It
+must contain no more than 30 classes. By default, all other
+{ref}/put-dfanalytics.html#dfa-supported-fields[supported fields] are included
+in the analysis and are known as _{feature-vars}_. The runtime and resources
+used by the job increase with the number of feature variables. Therefore, you
+can optionally include or exclude fields from the analysis. For more information
+about field selection, see the
+{ref}/explain-dfanalytics.html[explain data frame analytics API].
 
 
 [discrete]
 [[dfa-classification-supervised]]
 ==== Training the {classification} model
 
-{classification-cap} – just like {regression} – is a supervised {ml} process. It 
-means that you need to supply a labeled training dataset that has some 
-{feature-vars} and a {depvar}. The {classification} algorithm learns the 
-relationships between the features and the {depvar}. Once you’ve trained the 
-model on your training dataset, you can reuse the knowledge that the model has 
-learned about the relationships between the data points to classify new data. 
-Your training dataset should be approximately balanced which means the number of 
-data points belonging to the various classes should not be widely different, 
-otherwise the {classanalysis} may not provide the best predictions. Read 
-<<dfa-classification-imbalanced-classes>> to learn more.
-
+{classification-cap} – just like {regression} – is a supervised {ml} process.
+When you create the {dfanalytics-job}, you must provide a data set that contains
+the _ground truth_. That is to say, your data set must contain the {depvar} 
+and the {feature-vars} fields that are related to it. You can divide the data
+set into training and testing data by specifying a `training_percent`. By
+default when you use the
+{ref}/put-dfanalytics.html[create {dfanalytics-jobs} API], 100% of the eligible
```

Comment:
Could we please clarify what `eligible` documents are? For ex, I know we exclude documents if they have a field that has an array with more than one element (cc: @dimitris-athanasiou - could you please assist in getting this worded correctly). We don't have to define it in this paragraph, but it would be nice to link to a section that explains what documents are eligible.